modules/docs/arrow-docs/docs/docs/effects/io/README.md

```diff
@@ -40,6 +42,8 @@ All exceptions that would happen on the function parameter are automatically cap
 
 It runs the current `IO` asynchronously, calling the callback parameter on completion and returning its result.
 
+The operation will not yield a result immediately; ultimately to start running the suspended computation you have to evaluate that new instance using an unsafe operator like `unsafeRunAsync` or `unsafeRunSync` for `IO`.
+
 ```kotlin
```

Comment:
Any reason we are not using `ank` on these snippets?

Could we perhaps add an additional snippet to accompany the added docs?

```kotlin:ank
val io = IO<Int> { throw RuntimeException("Boom!") }
  .runAsync { result ->
    result.fold({ IO { println("Error") } }, { IO { println(it.toString()) } })
  }
```

The operation will ...

Reply:
Async operations fail to execute on Ank.